app/views/softdrinksindustrylevy/variations/retrieve_summary.scala.html

```diff
@@ -131,20 +133,9 @@ <h1 class="heading-large">@Messages("sdil.variation.heading")</h1>
                     </tr>
                 }
             }
-        } else {
-            @declaration_table("declaration", "production-sites") {
-                <tr>
-                    <td class="change-answer">
-                        <a href="@sdil.controllers.variation.routes.ProductionSiteVariationController.show()">
-                        @Messages("sdil.variation.add-link")
-                        </a>
-                    </td>
-                </tr>
-            }
         }
-    }
 
-    @if(variationData.isLiable) {
+    @if(!variationData.original.activity.voluntaryRegistration) {
```

Comment:
I don't think this does anything different from the code it replaces, but will need reverting if/when the user can change actor status.